shell/plugins/steve/subscribe.js

```diff
@@ -275,6 +303,12 @@ export const actions = {
       type, selector, id, revision, namespace, stop, force
     } = params;
 
+    if (['count', 'schema'].includes(type) && this.$workers[getters.storeName]?.mode === 'advanced') {
```

Comment:
Potentially. I don't think we should maintain a list of currently watched resources in the worker unless we want to post a message to it requesting a current list but this logic should probably be built into the worker itself rather than subscribe which shouldn't have to care overmuch about the worker once it's configured and instantiated.